Permanent/ViewControllers/SideMenuViewController.swift

```diff
@@ -73,6 +83,19 @@ class SideMenuViewController: BaseViewController<AuthViewModel> {
             tableView.isScrollEnabled = true
         }
     }
+    
+    func updateLeftSideMenu() {
+        tableView.beginUpdates()
+        NotificationCenter.default.post(name: Self.updateArchiveSettingsChevron, object: self, userInfo: nil)
+        if archiveSetingsWasPressed {
```

Comment:
archiveSetingsWasPressed this bool should probably be part of the viewModel. The ViewModel should also be the one posting the update notification